content/source/docs/enterprise/before-installing/reference-architecture/vmware.md

```diff
@@ -179,10 +179,10 @@ See [the Upgrades section](../../admin/upgrades.html) of the documentation. For
 
 ### Failure Scenarios
 
-VMWare hypervisor provides a high level of resilience in various cases
-of failure (at the server hardware layer through vMotion and at the network layer through virtual distributed
-networking.) In addition, having ESXi failover to a DR datacenter
-provides recovery in the case of a total data center outage See the Disaster Recovery section.
+VMware vSphere provides a high level of resilience in various cases
+of failure (at the server hardware layer through vSphere High Availablilty (HA) and at the network layer through virtual distributed
+switching.) In addition, having workloads failover to a DR datacenter with tools like VMware Site Recovery Manager or utilizing stretched clusters
+provides recovery in the case of a total data center outage. See the Disaster Recovery section.
```

Comment:
1. HA is the backing technology that is set on a vSphere cluster that restarts VMs on a designated physical ESXi host if the one that they are running on fails. vMotion only happens if DRS (Distributed Resources Scheduler) has a recommendation to migrate the workload to a host that has more available resources, a user manually migrates the workload, or the host that the workload was on was put in maintenance mode. 

2. I would say that HA is a requirement and this is enabled in most if not all vSphere clusters. 

3. I always refer to VMs as workloads but we can rephrase this to something else more specific if you would like. 

4. The callout to SRM or other products is really to have the customer be aware that they can use 3rd party tools to replicate the LUN where the VMs reside or the VMs themselves so that way they can have automated failover processes which replicate the running state of the workloads. This handles all of the replication.

Reply:
```suggestion
VMware vSphere provides a high level of resilience in various cases
of failure, such as at the server hardware layer through vSphere High Availability (HA) and at the network layer through virtual distributed
switching. In addition, employing tools such as VMware Site Recovery Manager or utilizing stretched clusters
can assist in recovery in the case of a total data center to support failover to a DR datacenter. See the Disaster Recovery section.
```

Reply:
Did a little rewording to soften the impression that an automatic DR failover can always be done just through the facilities and to better highlight add-on tools would be needed.